examples/data-sources/tls_certificate/data-source.tf

```diff
@@ -11,3 +11,7 @@ resource "aws_iam_openid_connect_provider" "example" {
   thumbprint_list = [data.tls_certificate.example.certificates[0].sha1_fingerprint]
   url             = aws_eks_cluster.example.identity[0].oidc[0].issuer
 }
+
+data "tls_certificate" "example_content" {
+  content = file("example.pem")
+}
```

Comment:
I'm not a fan of having 2 separate examples in the same example `.tf`, but if you are going this route, please add at least 1 line of comment for each to describe.

Reply:
Alternatively, if you want to split in 2 files, you can do that and then alter the template https://github.com/hashicorp/terraform-provider-tls/blob/main/templates/data-sources/certificate.md.tmpl to include both.

I had to do something similar for the examples of configuration of the Provider: it does allow to add a bit of verbal explanation between examples that it might be nice.

Reply:
@detro did this already :)